R/exclude.R

```diff
@@ -98,7 +98,11 @@ normalize_exclusions <- function(x, normalize_path=TRUE) {
       x[unnamed] <- Inf
     }
   }
-
+  # Paths to excluded files are specified relative-to the package- or
+  # directory-root (when running lint_dir or lint_package)
+  if (!is.null(dir_prefix)) {
+    names(x) <- file.path(dir_prefix, names(x))
+  }
   if (normalize_path) {
     x <- x[file.exists(names(x))]       # remove exclusions for non-existing files
```

Comment:
I think it would be best if
`x <- x[file.exists(names(x))]`
were outside the `if` statement.